src/main/scala/fs2/async/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
+
+import fs2.util.Catchable
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+package object async {
+
+
+  /**
+   * Create a new continuous signal which may be controlled asynchronously.
+   * Note that this would block any resulting processes (discrete, continuous) until any signal value is set.
+   */
+  def signalUnset[F[_]:AsyncExt,A]: Stream[F,mutable.Signal[F,A]] =
```

Comment:
To me `pipe` was kind of shortcut like doing pipe on every discrete/continuos stream than anything else. But I hear you on continuos signal that it may sound kind of strange, but is nothing else than sig.continuos.pipe(p1)

Unset signals are good for coordination (i.e. don't run anything until I say so, but be ready). Perhaps is ok to have them as Option[A] signals, as you may as well resume them.